pkg/cli/admin/upgrade/upgrade.go

```diff
@@ -340,20 +340,20 @@ func (o *Options) Run() error {
 				fmt.Fprintf(o.Out, "Upstream: %s\n", cv.Spec.Upstream)
 			}
 			if len(cv.Status.Desired.Channels) > 0 {
-				fmt.Fprintf(o.Out, "Channel: %s (available channels: %s)\n", cv.Spec.Channel, strings.Join(cv.Status.Desired.Channels, ", "))
+				fmt.Fprintf(o.Out, "Channel: %s (available channels: %s)\n\n", cv.Spec.Channel, strings.Join(cv.Status.Desired.Channels, ", "))
 			} else {
-				fmt.Fprintf(o.Out, "Channel: %s\n", cv.Spec.Channel)
+				fmt.Fprintf(o.Out, "Channel: %s\n\n", cv.Spec.Channel)
 			}
 		}
 
 		if len(cv.Status.AvailableUpdates) > 0 {
-			fmt.Fprintf(o.Out, "Available Updates:\n\n")
-			w := tabwriter.NewWriter(o.Out, 0, 2, 1, ' ', 0)
-			fmt.Fprintf(w, "VERSION\tIMAGE\n")
+			fmt.Fprintf(o.Out, "Recommended updates:\n\n")
+			w := tabwriter.NewWriter(o.Out, 14, 2, 1, ' ', 0)
```

Comment:
Not sure what `14` means here - is this padding?

Reply:
I have initially forgotten to add this change to the commit message.

The argument of `14` represents the `minwidth` ([minimal cell width including any padding](https://pkg.go.dev/text/tabwriter#NewWriter)). Which results in having the column have bigger minimal width than before. And since the ordinal versions have an approximate length of 6. The columns seem further apart.

But for longer version lengths, there won't be an unnecessary big empty space.
But a similar behavior could be achieved with fixed bigger `padding`.

Reply:
Ooh, I see, that makes sense

Lets add a code comment above this (`git blame` would show this, but the fewer WTF per code line the better)

Reply:
I have added the comment:
`// set the minimal cell width to 14 to have a larger space between the columns for shorter versions`

In the new commit https://github.com/openshift/oc/commit/30f708aadf0ead208b5d3888d80056a92a378036.